parts/k8s/kubernetesconfigs.sh

```diff
@@ -408,7 +408,7 @@ configAddons() {
 }
 
 configGPUDrivers() {
-    retrycmd_if_failure 30 1 120 sh $GPU_DEST/nvidia-drivers-$GPU_DV --silent --accept-license --no-drm --dkms --utility-prefix="${GPU_DEST}" --opengl-prefix="${GPU_DEST}" || exit $ERR_GPU_DRIVERS_START_FAIL
+    retrycmd_if_failure 3 1 1200 sh $GPU_DEST/nvidia-drivers-$GPU_DV --silent --accept-license --no-drm --dkms --utility-prefix="${GPU_DEST}" --opengl-prefix="${GPU_DEST}" || exit $ERR_GPU_DRIVERS_START_FAIL
```

Comment:
that's still 1 hour but should be okay for ARM timeout. Can the install really take up to 20 minutes?

Reply:
See the output above proving that 2 mins is too short. Would increasing timeout to a lower (but higher than 2) minute interval be better?

Reply:
how about 10 minutes?